frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -1166,20 +1166,29 @@ function FeatureFlagReleaseConditions({ readOnly }: FeatureFlagReadOnlyProps): J
                                                 {isPropertyFilterWithOperator(property) ? (
                                                     <span>{allOperatorsToHumanName(property.operator)} </span>
                                                 ) : null}
-                                                {[
-                                                    ...(Array.isArray(property.value)
-                                                        ? property.value
-                                                        : [property.value]),
-                                                ].map((val, idx) => (
-                                                    <span
-                                                        key={idx}
+
+                                                {property.type === 'cohort' ? (
+                                                    <a
+                                                        href={`/cohorts/${property.value}`}
```

Comment:
What do you think about making this a link that pops up a separate tab for that cohort vs redirecting it? 

Also it could be nice to use `urls.cohort()` here 🤗